qa/supply-chain/audits.toml

```diff
@@ -87,16 +158,95 @@ who = "Daira Hopwood <daira@jacaranda.org>"
 criteria = "safe-to-deploy"
 delta = "1.0.37 -> 1.0.41"
 
+[[audits.serde]]
+who = "Jack Grigg <jack@z.cash>"
+criteria = "safe-to-deploy"
+delta = "1.0.136 -> 1.0.143"
+notes = "Bumps serde-derive and adds some constructors."
+
+[[audits.serde_derive]]
+who = "Jack Grigg <jack@z.cash>"
+criteria = "safe-to-deploy"
+delta = "1.0.136 -> 1.0.143"
+notes = "Bumps syn, inverts some build flags."
+
 [[audits.syn]]
 who = "Daira Hopwood <daira@jacaranda.org>"
 criteria = "safe-to-deploy"
 delta = "1.0.91 -> 1.0.98"
 
+[[audits.thiserror]]
+who = "Jack Grigg <jack@z.cash>"
+criteria = "safe-to-deploy"
+delta = "1.0.30 -> 1.0.32"
+notes = "Bumps thiserror-impl, no code changes."
+
+[[audits.thiserror-impl]]
+who = "Jack Grigg <jack@z.cash>"
+criteria = "safe-to-deploy"
+delta = "1.0.30 -> 1.0.32"
+notes = "Only change is to refine an error message."
+
 [[audits.unicode-ident]]
 who = "Daira Hopwood <daira@jacaranda.org>"
 criteria = "safe-to-deploy"
 version = "1.0.2"
 
+[[audits.windows_aarch64_msvc]]
+who = "Jack Grigg <jack@z.cash>"
+criteria = "safe-to-run"
+version = "0.36.1"
+notes = """
+Adds a binary blob to the library search path, that contains a subset of
+the Windows SDK to avoid a direct dependency on the latter. See
+https://github.com/microsoft/windows-rs/pull/1217 for context. I did not
```

Comment:
I guess that's just par for the course for Windows :-/

Reply:
The binary blob contains libs we'd need to depend on either way; this just greatly reduces the compilation surface by only "bundl[ing] up all of the import libs needed to call any and all of the exported functions provided by the Windows API".